src/controller/org.controller/org.controller.js

```diff
@@ -486,6 +486,31 @@ async function updateUser (req, res, next) {
       }
     }
 
+    // check if org has user of same username already
+    if ((newUsername && newCnaShortname) && !returned) {
+      let duplicate_users = await User.find({ org_UUID: newUser.org_UUID, username: newUser.username })
+      if (duplicate_users.length) {
+        returned = true
+        logger.info('The user could not be updated because ' + newCnaShortname + ' CNA contains a user with the same name.')
+        return res.status(404).json({ message: 'The user could not be updated because ' + newCnaShortname + ' CNA contains a user with the same name.' })
```

Comment:
```suggestion
        return res.status(403).json({ error: 'DUPLICATE_USERNAME', message: `The user could not be updated because ${newCnaShortname} CNA contains a user with the same name: ${newUsername}` })
```